test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -2076,6 +2092,20 @@ Describe "Invoke-RestMethod tests" -Tags "Feature", "RequireAdminOnWindows" {
         $result.Output.Headers.Connection | Should -Be "Close"
     }
 
+    $httpVersions = @('1.1', '2')
+    foreach ($httpVersion in $httpVersions) {
+        It "Validate Invoke-RestMethod -HttpVersion $httpVersion" -Skip:(!$IsWindows) {
```

Comment:
This should probably be a `-TestCases` use rather than inside a `foreach` loop -- especially since that will make migrating to Pester 5 easier